src/component/actions/Action/Action.test.ts

```diff
@@ -9,16 +9,48 @@ import {
     actionInteractiveDivSpecId,
     actionBodySpecId
 } from './Action';
+import { FlowNode, SendMsg, StartFlow, SwitchRouter } from '../../../flowTypes';
 
-const config = require('../../../../assets/config');
-const colorsFlowResp = require('../../../../assets/flows/a4f64f1b-85bc-477e-b706-de313a022979.json');
+const config = require('../../../../__test__/assets/config');
 
-const {
-    results: [{ definition: { nodes: [sendMsgNode, , , sendMsgNode1, , , startFlowNode] } }]
-} = colorsFlowResp;
-const { actions: [sendMsgAction] } = sendMsgNode;
-const { actions: [sendMsgAction1] } = sendMsgNode1;
-const { actions: [startFlowAction] } = startFlowNode;
+const sendMsgAction: SendMsg = {
+    uuid: 'send_msg_action0',
+    type: 'send_msg',
+    text: 'Hello World!'
+};
+
+const sendMsgAction1: SendMsg = {
+    uuid: 'send_msg_action1',
+    type: 'send_msg',
+    text: 'Hello World!'
+};
+
+const sendMsgNode: FlowNode = {
+    uuid: 'send_msg_node',
+    actions: [sendMsgAction],
+    exits: []
+};
+
+const startFlowAction: StartFlow = {
+    uuid: 'start_flow_action',
+    type: 'start_flow',
+    flow_name: 'Flow to Start',
+    flow_uuid: 'flow_to_start'
+};
+
+const startFlowNode: FlowNode = {
+    uuid: 'start_flow_node',
+    actions: [startFlowAction],
+    exits: [],
+    router: {
+        type: 'switch',
+        operand: '@child'
+    } as SwitchRouter,
+    wait: {
+        type: 'flow',
+        flow_uuid: 'flow_to_start'
+    }
+};
```

Comment:
We should be following this pattern for action (and other unit level tests). Instead of pulling out bits and pieces from some complex flow definition, lets just define what we are going to test with inline and keep it simple. This reduces dependency entanglements and is much clearer at the small expense of some verbosity.

Reply:
agreed, was just working on some helpers for this sort of thing